src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/QueryPartitionStatistics.cs

```diff
@@ -16,6 +16,7 @@ internal class QueryPartitionStatistics
 
         public void Update(int partitionCount)
         {
+#pragma warning disable CA2002
```

Comment:
Can we add a comment here for why the rule is disabled?

Reply:
it's right there, one line below...

Reply:
Ah, yes. Sorry, got on a bit of autopilot.